lib/eventbrite.rb

```diff
@@ -0,0 +1,50 @@
+require "httparty"
+require_relative "eventbrite/event"
+require_relative "eventbrite/null_event"
+
+class Eventbrite
+  include HTTParty
+
+  base_uri "https://www.eventbriteapi.com/v3"
+
+  def upcoming_event
+    event_list.first
+  end
+
+  private
+
+  def event_list
+    if user_events_response.success? && !user_events_response["events"].empty?
+      user_events_response["events"].map do |event|
```

Comment:
What do you think about putting this events hash out into a method? You call it twice and having it pulled out would make it a bit more clear.